### PR TITLE
Redesign complete profile

### PR DESCRIPTION
**Pull request by cp-mayank**
- Redesign Complete profile progress 

complete_profile

## Summary by CodeRabbit

- **New Features**
  - Introduced a user profile completion progress indicator, showing users how complete their profiles are and suggesting actions to enhance it.
  - Added new profile completion options: add profile picture, personal details, and playing style.

- **Enhancements**
  - Updated profile completion descriptions to provide clearer guidance on enhancing profiles.

### CODE REVIEW

**Comment by cp-sidhdhi-p** [comment]
@cp-mayank Add tap event on card, navigate to edit profile if user taps on whole card

File: khelo/lib/ui/flow/profile/components/complete_profile_progress.dart [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+class _CompleteProfileProgressState
+    extends ConsumerState<CompleteProfileProgress> {
+  @override
+  Widget build(BuildContext context) {
+    if (widget.user?.profile_img_url != null &&
+        widget.user?.gender != null &&
+        widget.user?.player_role != null &&
+        widget.user?.batting_style != null &&
+        widget.user?.bowling_style != null) return const SizedBox();
```
Comment (cp-sidhdhi-p) [inline khelo/lib/ui/flow/profile/components/complete_profile_progress.dart:26]: if we're showing prompt for these fields only, then progress also should be measured for these fields only.

File: khelo/lib/ui/flow/profile/components/complete_profile_progress.dart [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    );
+  }
+
+  Widget _fieldStatus(String title) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 8.0),
+      child: Row(
+        children: [
+          Icon(
+            Icons.radio_button_off_rounded,
```
Comment (cp-sidhdhi-p) [inline khelo/lib/ui/flow/profile/components/complete_profile_progress.dart:83]: i think we need to show below Check mark with container Low color as jimmy sir said. image

File: khelo/lib/ui/flow/profile/components/complete_profile_progress.dart [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            color: context.colorScheme.textSecondary,
+          ),
+          const SizedBox(width: 16),
+          Text(
+            title,
+            style: AppTextStyle.subtitle2.copyWith(
```
Comment (cp-sidhdhi-p) [inline khelo/lib/ui/flow/profile/components/complete_profile_progress.dart:89]: this is body1 with text secondary color, could you please check at the other place like title and subtitle of this prompt, digita might have changed them.

File: khelo/lib/ui/flow/profile/components/complete_profile_progress.dart (hunk 1 above)
Comment (cp-mayank) [reply khelo/lib/ui/flow/profile/components/complete_profile_progress.dart:26]: No because we need to show the required field progress as well.

**Review by cp-sidhdhi-p — APPROVED** [review]

**Pull request merged**